docs/examples/example.ipynb

```diff
@@ -481,11 +481,11 @@
    "metadata": {},
    "source": [
     "### 5. Configure a modular resource\n",
-    "**NOTE:** *These instructions are for local development. Soon we will launch a `Registry` which will allow users to upload modules to app.viam.com directly. We will update the documentation with instructions on how to do that shortly.*\n",
+    "**NOTE:** *These instructions are for local development. If you are adding your module to the registry, follow [these instructions](https://docs.viam.com/extend/modular-resources/configure/) instead. Otherwise, continue with these instructions, which will show you how to configure the module locally.*\n",
```

Comment:
```suggestion
    "**NOTE:** *If you are adding your module to the registry, follow [these instructions](https://docs.viam.com/extend/modular-resources/configure/) instead. Otherwise, continue with these instructions, which will show you how to configure the module locally.*\n",
```